### ISSUE TITLE
"AnnData Operations" tool not indexed in usegalaxy.eu search

### ISSUE THREAD

**Issue by hexhowells** [usegalaxy-eu/usegalaxy-eu-tools#807]
The tool [AnnData Operations](https://toolshed.g2.bx.psu.edu/view/ebi-gxa/anndata_ops/a4774b7b2e85) doesn't appear when searched for in the [usegalaxy.eu](https://usegalaxy.eu) server. The tool however, does exist in the server and can be run via workflows. 

The tool does appear in the [singlecell.usegalaxy.eu](https://singlecell.usegalaxy.eu) subdomain.

**Comment by bgruening** [comment]
Yes, those tools are from EBI and have redundant functionality with already existing tools. At some point people decided to have those tools only on the subdomains. Should we revert this decision? Are the EBI tools still maintained?

ping @pavanvidem @nomadscientist

**Comment by pavanvidem** [comment]
Anndata operations tool is still used in some tutorials. Do we need it on the main server if it already appears on the subdomain. We will anyways promoting the subdomain.

**Comment by bgruening** [comment]
@pavanvidem @nilchia still relevant?

**Comment by nilchia** [comment]
It is still visible in Singlecell portal.
In main EU it is correctly hidden

**Comment by pavanvidem** [comment]
I think we covered most of the functionality of EBI tools in the IUC anndata tools. The best pathway is to replace the EBI tools with IUC tools wherever possible. 
I started an issue here https://github.com/galaxyproject/training-material/issues/7119 and started working on this here https://github.com/galaxyproject/training-material/pull/7120

We will continue there. This issue can be closed.
